conf/modules/call_sv_tiddit.config

```diff
@@ -20,6 +20,10 @@ process {
         ext.args   = '-p 6'
     }
 
+    withName: ".*CALL_STRUCTURAL_VARIANTS:CALL_SV_TIDDIT:BCFTOOLS_FILTER_TIDDIT" {
+        ext.args = { '-i \'FILTER="PASS"\'' }
```

Comment:
This can also be written as `'--apply-filters PASS'` or `-f PASS`. I kind of think it's a bit more explicit and clearer but it's really splitting hairs. 

Should we allow for `.` as well?